modules/message_dispatcher.lua

```diff
@@ -123,13 +124,8 @@ end
 -- @treturn number timeout Timeout for next message
 function fstream_mt.__index:GetMessage()
   local timespan = timestamp() - self.ts
-  if timespan > 5000 then
-    self.ts = self.ts + timespan - 1000
-    self.bytesSent = self.bytesSent / (timespan / 1000)
-    timespan = 1000
-  end
   if (self.bytesSent + self.chunksize) / (timespan / 1000) > self.bandwidth then
-    return nil, 200
+    return nil, 250
```

Comment:
@dboltovskyi please add description of 25- magic number

Reply:
@LuxoftAKutsan Done in b301857